### PR TITLE
Bugs fix misc tailwind

### PR DESCRIPTION
**Pull request by arpita-20**
Bug-fixes-
1-removed extra custom styles from css and used tailwind classes
2-removed unnecessary code from tailwind config file

### CODE REVIEW

File: app/components/results/GroupByTests.tsx [hunk 1]
```diff
@@ -47,8 +49,8 @@ const GroupByTests = () => {
           />
         </div>
 
-        <div className="rounded-lg bg-[#F9FAFB] shadow-table">
-          <div className="col-span-full grid grid-cols-10 rounded-lg border-[1px] border-solid border-borderColor bg-white">
+        <div className="rounded-lg shadow-base">
+          <div className="col-span-full grid grid-cols-10 rounded-lg border-[1px] border-solid border-gray-200 bg-white">
```
Comment (yash6102000) [inline app/components/results/GroupByTests.tsx:53]: remove border-[1px]..it's. by default

File: app/components/tests/CreateTestPreview.tsx [hunk 2]
```diff
@@ -110,7 +108,7 @@ const TestPreview = ({
           {selectedSections.map((section, i) => {
             return (
               <div className="flex items-center gap-4" key={section.id}>
-                <div className="w-44 min-w-[184px] text-base text-gray-500">
+                <div className="min-w-184 text-base text-gray-500">
                   {testsConstants.sectionText} {i + 1}
                 </div>
                 <div className=" flex max-w-2xl flex-1 items-center justify-between gap-6 rounded-lg border border-gray-300	py-3 px-4 text-gray-700">
```
Comment (yash6102000) [inline app/components/tests/CreateTestPreview.tsx:114]: remove extra space

File: app/components/tests/SelectSectionCard.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ const SelectSectionCard = ({
   return (
     <div
       id="section"
-      className={`flex w-sectionCard min-w-sectionCard flex-1 flex-col gap-2 rounded-lg border ${
+      className={`flex min-w-sectionCard flex-1 flex-col gap-2 rounded-lg border ${
         section.isSelected
           ? 'border-[3px] border-primary bg-white px-[18px] py-[22px]'
```
Comment (yash6102000) [inline app/components/tests/SelectSectionCard.tsx:59]: add this in tailwind config

File: app/components/tests/TestList.tsx [hunk 4]
```diff
@@ -57,15 +57,16 @@ const TestList = ({
         >
           {testsConstants.tests}
         </h2>
-        <Button 
-          className='px-5'
+        <Button
+          className="px-5"
           onClick={() => navigate('/tests/add-test')}
```
Comment (yash6102000) [inline app/components/tests/TestList.tsx:62]: remove extra space on line no 48 className="test-list-container flex h-full flex-col gap-6 "

File: tailwind.config.js [hunk 5]
```diff
@@ -10,35 +10,27 @@ module.exports = {
         primaryDisabled: '#A2A4D6',
         primaryOutlined: '#141649',
         primaryOutlinedDisabled: '#A2A4D6',
```
Comment (yash6102000) [inline tailwind.config.js:12]: why we are using same color A2A4D6 twice with diffrent names

File: app/components/tests/TestList.tsx [hunk 6]
```diff
@@ -79,7 +80,7 @@ const TestList = ({
         />
       </div>
       <div className="flex flex-1 flex-col rounded-lg pb-6 ">
-        <div className="rounded-b-0 flex items-center gap-3 rounded-t-md border border-b border-solid border-borderColor bg-tableHeader px-9 py-3 font-semibold shadow-table">
+        <div className="rounded-b-0 flex items-center gap-3 rounded-t-md border border-b border-solid border-gray-200 bg-tableHeader px-9 py-3 font-semibold shadow-base">
```
Comment (yash6102000) [inline app/components/tests/TestList.tsx:83]: add shadow on correct place

File: app/components/results/ExamPending.tsx [hunk 7]
```diff
@@ -13,8 +13,8 @@ const ExamPendingComponent = () => {
   return (
     <>
       {testData.length !== 0 ? (
-        <div className="bg-[#F9FAFB] pb-4">
-          <div className="col-span-full rounded-lg border border-solid border-gray-200 bg-tableHeader shadow-table">
+        <div className="pb-4">
+          <div className="col-span-full rounded-lg border border-solid border-gray-200 bg-gray-100 shadow-base">
             <div className="col-span-full grid grid-cols-10 gap-3 py-4 px-12">
               <span className="col-span-1 text-sm  font-semibold  text-gray-500">
```
Comment (yash6102000) [inline app/components/results/ExamPending.tsx:19]: remove extra space

File: app/components/tests/TestTableItem.tsx [hunk 8]
```diff
@@ -48,7 +48,10 @@ const TestTableItem = ({
     <>
       <div
         key={index}
-        className="test-table-list flex items-center border-b border-t border-gray-200 bg-white py-6 px-9">
+        className={`${
+          index === totalCount ? 'rounded-b-md' : ''
+        } test-table-list flex items-center border-b border-gray-200 bg-white py-6 px-9`}
```
Comment (yash6102000) [inline app/components/tests/TestTableItem.tsx:53]: remove extra space

File: app/components/tests/ChipGroup.tsx [hunk 9]
```diff
@@ -38,10 +41,13 @@ const ChipGroup = ({
...
                 return (
-                  <div className=" flex flex-col px-2 py-2" key={sect?.section?.id}>
+                  <div
+                    className="flex flex-col px-2 py-2"
+                    key={sect?.section?.id}
+                  >
                     <Menu.Item>
                       <span className="text-sm">
-                        <span className="truncate rounded-52 bg-bgcolor px-2 py-1">
+                        <span className="bg-bgcolor truncate rounded-52 px-2 py-1">
```
Comment (yash6102000) [inline app/components/tests/ChipGroup.tsx:50]: use tailwind class

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**